app/src/main/java/com/codesoom/assignment/HttpMethod.java

```diff
@@ -0,0 +1,19 @@
+package com.codesoom.assignment;
+
+public enum HttpMethod {
```

Comment:
잘 분리하셨어요. 이런 enum에는 rfc 문서 링크를 javadoc 주석으로 남겨주면 좋겠죠.
```suggestion
/**
 * HTTP 프로토콜 메소드를 정의합니다.
 * @link https://datatracker.ietf.org/doc/html/rfc2616#section-9
 */
public enum HttpMethod {
```